src/DateInput/index.js

```diff
@@ -56,21 +57,25 @@ const DateInput = ({
   };
 
   useEffect(() => {
+    if (disablePortal) {
+      flatpickrOptions.static = true;
+    }
     flatpickr(input.current, flatpickrOptions);
-  }, [flatpickrOptions, input]);
+  }, [flatpickrOptions, input, disablePortal]);
 
   return (
-    <Input {...props}>
-      <input
-        key={"nds-text"}
-        ref={input}
-        type="date"
-        required
-        placeholder={props.label}
-        data-testid={testId}
-        {...props}
-      />
-    </Input>
+    <div>
+      <Input {...props} label={props.label}>
```

Comment:
static mode of flatpickr does some weird things with the input placeholder - moving the label prop to the `Input` component makes things behave more normal regardless of the flatpickr mode